### PR TITLE
Fix php citations

### PR DESCRIPTION
**Pull request by rlaforge-va**
- Updated protos to version 2.0.0 (changed namespace to \vendasta\citations).
- composer.json now required grpc extension.
- now requiring exact version of grpc.

@vendasta/splinter

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -10,12 +10,12 @@ RUN mv composer.phar /usr/local/bin/composer
 
 COPY ./composer.json /citations-sdk/composer.json
 
+COPY ./src/sdk.php /citations-sdk/sdk.php
+COPY ./example/test_client.php /citations-sdk/example/test_client.php
```
Comment (etarasoff-va) [inline Dockerfile]: Test client should be relying on the stuff generated by the `composer install`, so it should come *after* that step.
Also, we've got a Dockerfile in the example folder for testing, so is this even necessary?

File: Dockerfile [hunk 2]
```diff
@@ -10,12 +10,12 @@ RUN mv composer.phar /usr/local/bin/composer
...
+COPY ./src/google/timestamp.php /citations-sdk/timestamp.php
+COPY ./src/citations.php /citations-sdk/citations.php
+
 WORKDIR /citations-sdk
 RUN composer install  # This is where the Vendor folder gets created
```
Comment (etarasoff-va) [inline Dockerfile]: Don't we need a `RUN composer dump-autoload -o` here too, or has that changed?

File: Dockerfile (hunk 1 above)
Comment (jfingler-va) [inline Dockerfile]: This one looks better than the other one. The example dockerfile doesn't look right at all, and we don't have one in the example folder in our listing sdk, so we should use the same convention I suppose.

**Comment by eschick-va** [comment]
`COMPLETE-OK`

**Pull request merged**